.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
```

Comment:
Had to add this rule, because otherwise, my editor makes all the files I touched to use 4 spaces for tabs :)

Reply:
It's somewhat weird for me that the solution requires a config file in the repo. Shouldn't this be handled as an IDE config?

Where are the tabs used?